python/paddle/fluid/layers/nn.py

```diff
@@ -5281,3 +5282,74 @@ def crop(x, shape=None, offsets=None, name=None):
         outputs={'Out': out},
         attrs=None if len(attrs) == 0 else attrs)
     return out
+
+
+def rank_loss(label, left, right, name=None):
+    """
+    **RankLoss layer for RankNet**
+
+    (http://icml.cc/2015/wp-content/uploads/2015/06/icml_ranking.pdf).
+    RankNet is a pairwise ranking model with
+    one training sample consisting of a pair of doc A and B, and the label P
+    indicating that A is ranked higher than B or not:
+    
+    P = {0, 1} or {0, 0.5, 1}, where 0.5 means no information about the rank of
+    the input pair.
+    
+    The RankLoss layer takes three inputs: Left (o_i), Right (o_j) and Label
```

Comment:
The RankLoss layer -> This layer
Left (o_i), Right (o_j) and Label -> `left` (o_i), `right` (o_j) and `label`

Reply:
Done